src/vs/base/browser/ui/scrollbar/scrollableElement.ts

```diff
@@ -331,8 +331,15 @@ export abstract class AbstractScrollableElement extends Widget {
 		// console.log(`${Date.now()}, ${e.deltaY}, ${e.deltaX}`);
 
 		if (e.deltaY || e.deltaX) {
-			let deltaY = e.deltaY * this._options.mouseWheelScrollSensitivity;
-			let deltaX = e.deltaX * this._options.mouseWheelScrollSensitivity;
+			let deltaY = e.deltaY;
+			let deltaX = e.deltaX;
+			if (platform.isWindows) {
```

Comment:
Are you sure this only affects Windows?

Reply:
I don't know about linux, but it doesn't seem to affect macOS